plugins/services/src/js/utils/QuotaUtil.ts

```diff
@@ -58,10 +58,15 @@ export function populateResourcesFromRole(
       quota.gpus.limit = getValue(groupQuota.limit.gpus);
     }
     if (groupQuota.consumed) {
-      quota.cpus.consumed = getValue(groupQuota.consumed.cpus);
-      quota.memory.consumed = getValue(groupQuota.consumed.mem);
-      quota.disk.consumed = getValue(groupQuota.consumed.disk);
-      quota.gpus.consumed = getValue(groupQuota.consumed.gpus);
+      quota.cpus.consumed = getValue(groupQuota.consumed.cpus) || 0;
```

Comment:
In testing consumed is empty until we have tasks with the role, but I still wanted to show 0 so I added this code to always default consumed values to 0 if we have a role.